docs/Course Wiki/ECE Course Offerings/ECE476.md

```diff
@@ -2,41 +2,58 @@
 
-## When to Take It
+This modeling all builds up to the formulation of the power flow problem. When designing and analyzing large electrical system operation, it is vital to understand the voltages, currents, real power, and reactive power values at every bus. The problem is that we may only know a few of these quantities like a singular generator voltage or the impedance of a transmission line, so it is no easy task to figure out all unknowns. To solve for all of the unknown quantities within a power system, we have to employ nonlinear system solving methods that are iterative in nature and are similar to the Newton-Raphson method. The class delves very deep into the formulation of these methods, all of the assumptions needed, and the costs and benefits of each technique. Students get to write their own programs to solve basic power flow problems using methods such as Newton's method, DC power flow, decoupled power flow, and fast decoupled power flow. 
 
+Next students are introduced to the idea of economic dispatch (tertiary control) and learn about cost function modeling of generators. Control and mathematical optimization concepts are introduced as students learn about the economics behind power generation within our modern day grid. 
 
+Unbalanced system operation is introduced to students through the method of symmetrical components. Students learn about the different types of faults that can occur within a three phase system (single line-ground, line-line, double line-ground, open, balanced three-phase-ground) and how to model them using linear algebra techniques as well as basic circuit analysis. 
 
+Finally, students go back to generator modeling and dive deep into transient stability during fault conditions. The governing differential equations for generators are studied in depth and rigorously analyzed to build up to the equal area criterion for transient stability analysis. Parallels are drawn between generator transients and mass-spring systems to help students conceptualize the difficult topics. 
+## Prerequisites
+-[ECE330](ECE330.md)
+This class heavily relies on content from that class. There are many topical prerequisites for this class as it relies on familiarity of differential equations, matrix algebra, and computer programming. 
```

Comment:
Can you mention concepts that are taught in 330 that are also used in 476, like 3 phase circuits, complex power, transformers, etc